bin/atomizer

```diff
@@ -47,11 +51,15 @@ if (process.argv.length <= 2) {
 
 var atomizer = new Atomizer({ verbose: !!program.verbose });
 
+function warn(output) {
+  if (!program.quiet) console.warn(outout);
```

Comment:
typo

Reply:
`console.warn` can take N arguments, which is used in one place.  Should just take `arguments` and pass via `apply()`

Reply:
Correction - we're not actually using extra args currently.  I read line 96 wrong!

Reply:
:) I think it no harm just take `arguments` so if later we want to use it will be easier